l1-contracts/contracts/governance/PermanentRestriction.sol

```diff
@@ -209,8 +209,18 @@ contract PermanentRestriction is Restriction, IPermanentRestriction, Ownable2Ste
     /// @notice Checks if the `msg.sender` is an admin of a certain ZkSyncHyperchain.
     /// @param _chain The address of the chain.
     function _isAdminOfAChain(address _chain) internal view returns (bool) {
```

Comment:
@StanislavBreadless is this the preferable implementation??

Reply:
the one in `kl/sync-layer-reorg` is preferrable